Solutions/Vectra/Hunting Queries/VectraMaliciousToolFileCopy.yaml

```diff
@@ -0,0 +1,31 @@
+id: e10782ff-f379-4b59-bccf-4724b73e21f9
+name: Vectra AI - Malicious Tools File Copy 
+description: |
+  'Query searches for suspicious malicious file names copy over SMB.
+  Metadata required = metadata_smbfiles'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: AIVectraStream
+    dataTypes:
+      - VectraStream
+tactics:
+  - CredentialAccess
+  - LateralMovement
+  - Collection
+relevantTechniques:
+  - T1557.001
```

Comment:
Wondering if it also makes sense to have Technique T1021.002 here?

Reply:
Indeed, not pointing to the right tactic. it should map to T1570